core/src/main/scala/akka/persistence/jdbc/schema/javadsl/SchemaUtils.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2014 - 2019 Dennis Vriend <https://github.com/dnvriend>
+ * Copyright (C) 2019 - 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.jdbc.schema.javadsl
+
+import java.util.concurrent.CompletionStage
+
+import akka.Done
+import akka.actor.ClassicActorSystemProvider
+import akka.persistence.jdbc.schema.scaladsl
+import scala.compat.java8.FutureConverters._
+import scala.concurrent.Future
+
+import akka.dispatch.Dispatchers
+import akka.persistence.jdbc.db.SlickDatabase
+import akka.persistence.jdbc.db.SlickExtension
+import slick.jdbc.JdbcBackend.Database
+
+object SchemaUtils {
+
+  /**
+   * JAVA API:
+   *
+   * For testing purposes the journal and snapshot tables can be created programmatically.
+   * For production it's recommended to create the table with DDL statements before the system is started.
```

Comment:
I think it'd be great to have a dedicated module for these tools. Getting them in production code, out of the box looks weird.

Reply:
maybe a new module (jar artifact) is overkill, but just by placing them in a `testkit` package would clarify expectations

Reply:
I will move it to `testkit` although I don't think it's necessary. 

I don't think it's such a bad thing to not have it as is. We do the same in projection. The `createIfNotExists` methods are on the respective Projection companions. 

We have the scaladoc saying we don't recommend it. The method name is super explicit about it, it's `dropAndCreate`.